src/components/FeaturePanel/Runways/SuccessRunways.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import React from 'react';
+import { Runway as RunwayType } from './loadRunways';
+import { t } from '../../../services/intl';
+
+const fmtRunwaySize = (runway: RunwayType) =>
+  `${runway.length || '?'} - ${runway.width || '?'}`;
+
+const Runway: React.FC<{ runway: RunwayType; showSize: boolean }> = ({
+  runway,
+  showSize,
+}) => (
+  <TableRow>
+    <TableCell>
+      <Link href={`/${runway.type}/${runway.id}`}>
+        {runway.ref || 'Unknown'}
+      </Link>
+    </TableCell>
+    {showSize && <TableCell>{fmtRunwaySize(runway)}</TableCell>}
+    {/* TODO: Use id tagging presets to show the surface in the users language */}
+    <TableCell style={{ textTransform: 'capitalize' }}>
+      {runway.surface || 'Unknown'}
+    </TableCell>
+  </TableRow>
+);
+
+interface RunwayProps {
```

Comment:
Please always use `type Props` intead of interface. The issue is if you have two interfaces with same name in the project, typescript will merge them into one type.

Reply:
Wow thanks I didn't know that!